### ISSUE TITLE
MAC addresses are not returned by the `/v1/device` endpoint on the Intel NUC

### ISSUE THREAD

**Issue by Ereski** [balena-io/balena-supervisor#1544]
Supervisor version: 11.4.10

I checked these on my NUC and there should be 2 valid interfaces to get MAC addresses from: https://github.com/balena-io/balena-supervisor/blob/master/src/lib/mac-address.ts#L33-L35. Instead, `/v1/device` doesn't even contain a `mac_address` property.

**Comment by 20k-ultra** [comment]
@pipex does your NUC experience this issue on the latest SV ?

**Comment by pipex** [comment]
@20k-ultra it looks like that change was introduced in 11.8.4

https://github.com/balena-io/balena-supervisor/commit/1b91ef3405d1d2013cbdd6381ed69256c6432d18

**Comment by 20k-ultra** [comment]
yeah, that's a good point. The version mentioned here is not affected by the source linked.

**Comment by 20k-ultra** [comment]
I wonder then if v11.8.4+ has this issue. I'll have to flash my NUC but if anyone wants to, feel free to update their NUC and give it a try.
